pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -41,42 +41,37 @@
 
 from ._adjoint_jacobian import LightningGPUAdjointJacobian
 from ._measurements import LightningGPUMeasurements
+from ._mpi_handler import LightningGPU_MPIHandler
```

Comment:
We shouldn't use `LightningGPU_MPIHandler`. 

We can either rename it to
```suggestion
from ._mpi_handler import MPIHandler
```
Or,
```suggestion
from ._mpi_handler import LightningGPUMPIHandler
```